packages/input-iban/src/formatters.js

```diff
@@ -8,7 +8,7 @@ import { friendlyFormatIBAN } from '@bundled-es-modules/ibantools';
  */
 export function formatIBAN(modelValue) {
   // defensive code because of ibantools
-  if (modelValue === '') {
+  if (!modelValue) {
```

Comment:
Ideally, this should be a fix instead of a chore

Reply:
Agreed